src/TextUI/Configuration/Merger.php

```diff
@@ -195,63 +195,66 @@ public function merge(CliConfiguration $cliConfiguration, XmlConfiguration $xmlC
         $coverageTextShowUncoveredFiles = false;
         $coverageTextShowOnlySummary    = false;
         $coverageXml                    = null;
+        $coverageFromXmlConfiguration   = true;
 
-        if (!($cliConfiguration->hasNoCoverage() && $cliConfiguration->noCoverage())) {
```

Comment:
It is worth reviewing this without whitespace diffs:
https://github.com/sebastianbergmann/phpunit/pull/4756/files?diff=unified&w=1